sdcm/logcollector.py

```diff
@@ -810,7 +810,9 @@ class ScyllaLogCollector(LogCollector):
                     CommandLog(name='coredumps.info',
                                command='sudo coredumpctl info'),
                     CommandLog(name='io-properties.yaml',
-                               command='cat /etc/scylla.d/io_properties.yaml')
+                               command='cat /etc/scylla.d/io_properties.yaml'),
+                    CommandLog(name='dmesg.log',
+                               command='dmesg -P')
```

Comment:
add a comma at the end so next time you won't need to change additional line

Reply:
Got it.